cluster/plan.go

```diff
@@ -392,6 +393,12 @@ func (c *Cluster) BuildKubeControllerProcess(host *hosts.Host, serviceOptions v3
 		}
 	}
 
+	for arg, value := range c.Services.KubeController.ExtraArgsArray {
+		CommandArgs[arg] = value
+	}
+
+	Command = appendArgs(Command, CommandArgs)
+
 	for arg, value := range CommandArgs {
 		cmd := fmt.Sprintf("--%s=%s", arg, value)
 		Command = append(Command, cmd)
```

Comment:
It looks like the above call to appenArgs has done the work, so we can drop this section.